build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -197,7 +197,7 @@ CONFIG_BOOT_RUNFROMFLASH=y
 # Boot Memory Configuration
 #
 CONFIG_RAM_KREGIONx_START="0x10005000,0x02000000,"
-CONFIG_RAM_KREGIONx_SIZE="487424,3145728,"
+CONFIG_RAM_KREGIONx_SIZE="466944,3145728,"
```

Comment:
487,424-466,944 = 20,480 = 20KB != 16KB

Reply:
sorry it has 4KB reserved for non secure callable region as well. so total 20KB has to be excluded.

BD_RAM_NS (rwx)  :      ORIGIN = 0x10005000, LENGTH = 0x10077000 - 0x10005000 /* MAIN RAM NS: 456K */
BD_RAM_S (rwx)  :       ORIGIN = 0x10077000, LENGTH = 0x1007B000 - 0x10077000   /* MAIN RAM S: 16K */
BD_RAM_NSC (rwx)  :  ORIGIN = 0x1007B000, LENGTH = 0x1007C000 - 0x1007B000   /* MAIN RAM NSC: 4K */

Reply:
Could you add this content in commit description? (I don't mean PR description)
```
BD_RAM_NS (rwx) : ORIGIN = 0x10005000, LENGTH = 0x10077000 - 0x10005000 /* MAIN RAM NS: 456K /
BD_RAM_S (rwx) : ORIGIN = 0x10077000, LENGTH = 0x1007B000 - 0x10077000 / MAIN RAM S: 16K /
BD_RAM_NSC (rwx) : ORIGIN = 0x1007B000, LENGTH = 0x1007C000 - 0x1007B000 / MAIN RAM NSC: 4K */
```

Reply:
added in commit description

Reply:
Sorry for late reply. Don't we need to modify this for all of defconfigs?